connector-packager/tests/test_xsd_validator.py

```diff
@@ -65,3 +65,21 @@ def test_validate_single_file(self):
         logging.debug("test_validate_single_file xml violations:")
         for violation in xml_violations_buffer:
             logging.debug(violation)
+
+    def test_validate_vendor_prefix(self):
+
+        test_file = TEST_FOLDER / "modular_dialog_connector/connectionFields.xml"
+        file_to_test = ConnectorFile("connectionFields.xml", "connection-fields")
+        xml_violations_buffer = []
+
+        self.assertTrue(validate_single_file(file_to_test, test_file, xml_violations_buffer),
+                        "Valid XML file not marked as valid")
+
+        print("\nTest malformed xml. Throws XML validation error.")
+        test_file = TEST_FOLDER / "broken_xml/connectionFields.xml"
+        self.assertFalse(validate_single_file(file_to_test, test_file, xml_violations_buffer),
+                         "XML file that doesn't follow valid name values marked as valid")
```

Comment:
This sentence is a bit unclear, took me a few tries to figure out what it was trying to say. How about:

```suggestion
                         "XML file with invalid name values marked as valid")
```

This is a bit more explicit on what has failed when skimming the results.